cmd/antrea-controller/controller.go

```diff
@@ -91,6 +92,10 @@ func run(o *Options) error {
 	if err != nil {
 		return fmt.Errorf("error creating K8s clients: %v", err)
 	}
+	// Create custom admission webhooks to label Namespaces with name if enabled in config.
+	if o.config.EnableNamespaceLabelMutator {
+		createNamespaceLabelMutatorWebhook(client)
```

Comment:
i wonder if this is better served as a controller which reconciles the webhook generation.. but also think it could be over engineering

Reply:
I forgot if we discussed what will happen when Antrea is deleted?

Reply:
this is a valid concern and the side effect could be namespace creations fails which is not good.. im more leaning towards reverting this commit to the manifest approach. especially considering 1.21 we will get the namespace name label in upstream itself.

Reply:
Reverting to manifest sounds good to me.